client.js

```diff
@@ -68,7 +70,7 @@ class Client extends EventEmitter {
     // Remove trailing slash from trackers to catch duplicates
     announce = announce.map(announceUrl => {
       announceUrl = announceUrl.toString()
-      if (announceUrl[announceUrl.length - 1] === '/') {
+      if (announceUrl[announceUrl.length - 1] === '/index.js') {
```

Comment:
Is this an automated change?

Reply:
yeah, linter auto-fixed that, didn't even notice

Reply:
@DiegoRBaquero fixed